README.md

```diff
@@ -1,3 +1,14 @@
+** Helm2 Users
```

Comment:
Not sure what formatting you're going for here, but it isn't working. Maybe this:
```suggestion
## Helm2 Users
```